src/unit.cpp

```diff
@@ -35,10 +37,13 @@ Unit::Unit(const Unit::Id id) {
     m_unit = m2 * m2;
     break;
   case Unit::Id::Counts:
-    m_unit = counts;
+    m_unit = counts * none;
```

Comment:
What is the purpose of the `* none`?

Reply:
This is a "trick" we found to overcome some subtleties of working with heterogeneous unit systems in `boost::units`: We are combing SI units with our own, and the two are considered independent unless you convert explicitly. Therefore, in operations like `(counts * m) / m`, `boosts` is not cancelling the `m` as expected --- you get `counts * dimensionless`.

Explicitly putting a factor `dimensionless` (`none`) into all our non-SI units was the easiest way we found to avoid writing special-case handling in all operations (which would attempt to remove the `dimensionless` factor manually.

I'll put a comment in the code.